### DISCUSSION TITLE
Language Settings?

### DISCUSSION THREAD

**Discussion by Ph0eN1x1917** [openscopeproject/TrguiNG#211] [Q&A]
Just logged into my Synology NAS and opened 'Transmission' but got this TrguiNG interface instead. Seems to be in Chinese language and I can't find the language preference for the life of me. Any advice?

**Comment by qu1ck** [comment]
TrguiNG is English only for now, it sounds like you have installed one of Chinese forks on your NAS. I don't know anything about synology so can only give general advice, check your installed packages.

**Comment by Ph0eN1x1917** [comment]
That’s interesting! It would be an arm package that when I quickly looked
for it, was unable to find it for Synology.

Best Regards,

Jeff Hamilton

c:727-239-6797

Computer Medic Repair & Service LLC <http://www.computermedic.me/>


<https://www.facebook.com/computermedic1>

**Reply by qu1ck** [reply]
I would check transmission package too, maybe you got one that bundles Chinese TrguiNG with it.

**Comment by akela76** [comment]
Hello. I really liked your program. 
Will there be support for languages other than English? I am interested in the Russian language.

**Reply by qu1ck** [reply]
Yes, there will be translations support, can't promise it will be soon.

**Reply by akela76** [reply]
Thanks
